src/com/yetanalytics/lrs_admin_ui/views/status.cljs

```diff
@@ -0,0 +1,227 @@
+(ns com.yetanalytics.lrs-admin-ui.views.status
+  (:require
+   [reagent.core :as r]
+   [re-frame.core :refer [subscribe dispatch]]
+   [clojure.string :as cs]
+   [com.yetanalytics.lrs-admin-ui.functions :as fns]
+   [com.yetanalytics.lrs-admin-ui.views.status.vis :as vis]
+   [goog.string :refer [format]]
+   [goog.string.format]))
+
+(defn big-number
+  [vis-key label sub-qvec]
+  (let [value @(subscribe sub-qvec)]
+    [:div.big-number
+   [:div.big-number-value
+    {:class (if @(subscribe [:status/loading? vis-key])
+              "spinner"
+              "")}
+    (or value " ")]
+   [:div.big-number-label
+    label]]))
+
+(defn timestamp
+  [vis-key label sub-qvec]
+  (let [value @(subscribe sub-qvec)]
+    [:div.timestamp
+     [:div.timestamp-label
+      label]
+     [:div.timestamp-value
+      {:class (if @(subscribe [:status/loading? vis-key])
+                "spinner"
+                "")}
+      (or value " ")]]))
+
+(defn refresh-button
+  []
+  [:div.status-refresh-button
+   [:input.btn-blue-bold
+    {:type "button"
+     :value "REFRESH"
+     :on-click #(dispatch [:status/get-all-data])}]])
+
+(defn title-loading-spinner
+  [vis-key]
+  [:span.vis-title-loading-spinner
+   {:class (if @(subscribe [:status/loading? vis-key])
+             "spinner"
+             "")}])
+
+(defn platform-pie
+  []
+  [:div.vis-pie
+   [:h4 "PLATFORMS"
+    [title-loading-spinner "platform-frequency"]]
+   [vis/pie
+    {:theme (aget vis/theme "material")
+     :data @(subscribe [:status.data/platform-frequency])
```

Comment:
Perhaps you can make a special case when the platform frequency map is empty. Right now when there are no statements in the LRS, you just get a whole bunch of blank space at the bottom.

Reply:
https://github.com/yetanalytics/lrs-admin-ui/pull/39/commits/8af71bc5b480101358e018491ff410f6bf6207cd